tracer/samples/AzureFunctionsWithAgentLessLogging/FunctionWithLogging/run.ps1

```diff
@@ -0,0 +1,133 @@
+<#
+.SYNOPSIS
+    A script that allows the user to run C# sample applications to demonstrate
+    the setup and usage of Datadog's .NET Tracer log collection and log trace ID injection in a function
+.PARAMETER AppDirectory
+    The application directory - defaults to the location of this script and need not be set
+    by the user unless samples are moved around.
+.PARAMETER ApiKey
+    The Datadog API key (only required when -Agentless is set)
```

Comment:
nitpick but the `(only required when -Agentless is set)` can be removed for this

```suggestion
    The Datadog API key
```